MapboxCoreNavigation/RouteController.swift

```diff
@@ -211,6 +223,7 @@ open class RouteController: NSObject {
 
     deinit {
         suspendLocationUpdates()
+        sendCancelEvent(rating: endOfRouteStarRating, comment: endOfRouteComment)
```

Comment:
@bsudekum were you able to verify that a telem event gets sent on cancel even with the call happening during deinit?

Reply:
Yep, confirmed it is sent.